libs/blocks/global-navigation/global-navigation.js

```diff
@@ -204,6 +204,12 @@ const closeOnClickOutside = (e) => {
   }
 };
 
+const getIetfLocale = (ietfLocale) => {
+  const nonStandardLocaleMap = { no_NO: 'nb_NO' };
+  if (nonStandardLocaleMap[ietfLocale]) return nonStandardLocaleMap[ietfLocale];
+  return ietfLocale;
```

Comment:
Nit: could we not combine the two lines in `return nonStandardLocaleMap[ietfLocale] || ietfLocale `?

Reply:
Good suggestion 👍 Made the change